content/guides/tinacms/react-data-fetching/guide.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Querying Tina Content Client-side in React
```

Comment:
```suggestion
title: Querying Tina Content Client-side at Runtime
```
- just a suggestion - this feels like it is more about runtime / client-side than it is about React. Like you could use the same technique in Nextjs, for example